Form/Type/BaseStatusType.php

```diff
@@ -56,11 +56,7 @@ public function __construct($class, $getter, $name, $flip = false)
      */
     public function getParent()
     {
-        // NEXT_MAJOR: Remove ternary and keep 'Symfony\Component\Form\Extension\Core\Type\ChoiceType'
-        // (when requirement of Symfony is >= 2.8)
-        return method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
-            ? 'Symfony\Component\Form\Extension\Core\Type\ChoiceType'
-            : 'choice';
+        return 'Symfony\Component\Form\Extension\Core\Type\ChoiceType';
```

Comment:
Consider bumping php version to 5.5 or 5.6 also and use `ChoiceType::class` notation